packages/mermaid/src/diagrams/info/infoParser.ts

```diff
@@ -0,0 +1,21 @@
+import type { Info } from 'mermaid-parser';
+import { parse } from 'mermaid-parser';
+
+import { log } from '../../logger.js';
+import type { ParserDefinition } from '../../diagram-api/types.js';
+import type { InfoDB } from './infoTypes.js';
+import { db } from './infoDb.js';
+import { populateCommonDb } from '../common/populateCommonDb.js';
+
+function populateDb(ast: Info, db: InfoDB) {
+  populateCommonDb(ast, db);
+  ast.version && db.setVersion(ast.version);
```

Comment:
The AST would not have the correct version. And we won't (most probably) release a new package of parser with every mermaid release. So the infoParser should always set the version from mermaid's package.json. 

```suggestion
  import { verison } from '../../../../package.json';
  // inside populateDB
  db.setVersion(verison);
```

Reply:
> We would need to setup up versioning in package.json, so that is is apparent for users when a new parser is required.

The parser version is not a concern for users as the right one will be included as part of mermaid automatically. 

The only subset of users who might be concerned about parser version is someone who is planning to use the AST or LSP separately, outside of mermaid. Linters, Code highlighting, etc.

Reply:
Done 5fbf63eeb63861196bc903ce887f327bb8757024